index.html

```diff
@@ -102,7 +103,7 @@ <h2>I'm interested in being a...</h2>
                     <h1 class="tagline">HackNC 2023</h1>
                     <h2 class="date">October 27-29, 2023</h2>
                    <!-- <p class = "update"> Event Ended ...Stay tune for 2022 </p> -->
-                    
+                   
                     <!-- <button class="live-button hide-on-med-and-down"><a href="https://www.hacknc.com/live/">→ Event Live</a></button> -->
                     <!-- <button class="involve-button hide-on-med-and-down"><a href="https://forms.gle/cqkwFZum1BihNbpKA">2022 Hacker Interest form</a></button> -->
```

Comment:
Can you actually remove this commented-out code from the previous PR? We don't need it for now and can always reference previous commits if we want to add it back in eventually